### PR TITLE
Cre 873

### PR DESCRIPTION
**Pull request by vivek02172003**
**What is the purpose of this change?**
To build the Prometheus Exporter into a docker image

**How is this accomplished?**
This is accomplished by designing a MPB that builds a docker image, tars the image, and places it into loads

**Anything reviews should focus on/be aware of?**
Ensure to also review the pipeline script in Jenkins http://jenkins2/job/prometheus-exporter-build/configure

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,10 +8,13 @@ ENV GOARCH=amd64
 WORKDIR /go/src/app
 COPY . .
 
-RUN go get -d -v ./... \
- && go install -v ./... \
- && go test -short ./... \
- && go build \
+RUN go get -d -v ./...
```
Comment (GreenRover) [inline Dockerfile:11]: each RUN creates a docker layer. That is expensive.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+currentBuild.description = 'Cause: version bump'
```
Comment (GreenRover) [inline Jenkinsfile:1]: This file is not needed. We build with docker hub scripts. There is no need for local builds.

File: goversion/version.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// pubsubplus-go-client
```
Comment (GreenRover) [inline goversion/version.go:1]: What is the use case of this file?

**Review by GreenRover — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (GreenRover) [reply Dockerfile:11]: https://vsupalov.com/docker-image-layers/

**Comment by vivek02172003** [comment]
This pull request was meant to target https://github.com/SolaceDev/pubsubplus-prometheus-exporter. Sorry for any inconveniences.

**Comment by GreenRover** [comment]
What is: https://github.com/SolaceDev/pubsubplus-prometheus-exporter
Are you from solace and anting to play around?
Or where should the journey go to?

**Pull request closed without merging**